package.json

```diff
@@ -64,6 +64,7 @@
     "lint": "npm run --silent lint:js && npm run --silent lint:jenkinsfile && npm run --silent lint:package-names",
     "lint:ci": "npm run --silent lint:js:ci && npm run --silent lint:jenkinsfile && npm run --silent lint:package-names",
     "lint:commitmsg": "commitlint -e $GIT_PARAMS",
+    "lint:errors": "npm run lint | grep --color -e '\\serror\\s' || true",
     "lint:eslint": "eslint --ignore-pattern 'packages/node_modules/generator-ciscospark/generators/*/templates/**' --ignore-pattern 'packages/node_modules/**/umd/**' --ignore-pattern 'docs/**' --ignore-path .gitignore",
```

Comment:
We could just add `--silent` to quiet warnings via command line: https://eslint.org/docs/user-guide/command-line-interface#--quiet